README.md

```diff
@@ -41,10 +41,18 @@ The priority is applied as follows:
 
 The order of objects with equal (implicit) priority is undefined.
 
+## GlobalTracer
+
+If the [opentracing-util] library is detected and a [`GlobalTracer`][globaltracer] 
```

Comment:
(I am not nearly sophisticated enough as a JVM programmer to have an opinion about the classloader-level stuff, but – for what it's worth – the semantics you describe here in English sound right. Thanks!)